docker-compose.yml

```diff
@@ -8,6 +8,7 @@ services:
       - 8108:8108 # typesense server
     volumes:
       - ./typesense-data:/data
+<<<<<<< Updated upstream
```

Comment:
@jaeyson Merge conflict?

Reply:
oops, fixed conflict now. Just wanted to know @kianmeng what are the next steps here.